community/include/community.hpp

```diff
@@ -411,26 +441,54 @@ CONTRACT community : public contract
         map<name, uint64_t> voters_detail;
         uint64_t primary_key() const { return cadidate.value; }
         double by_voted_percent() const { return -voted_percent;}
+
+        EOSLIB_SERIALIZE( pos_candidate, (cadidate)(voted_percent)(voters_detail));
     };
     typedef eosio::multi_index<"poscandidate"_n, pos_candidate,indexed_by<"byvoted"_n, const_mem_fun<pos_candidate, double, &pos_candidate::by_voted_percent>>> poscandidate_table;
 	/*
 	* global singelton table, used for position id building
 	* Scope: self
 	*/
-	TABLE global{
+	TABLE global {
 		  
 		global(){}
 		uint64_t posproposed_id	= 0;
-
-		EOSLIB_SERIALIZE( global, (posproposed_id) )
+        name community_creator_name = "c"_n;
+        name cryptobadge_contract_name = "badge"_n;
+        name token_contract_name = "tiger.token"_n;
+        symbol core_symbol = symbol(symbol_code("CAT"), 4);
+        symbol ramcore_symbol = symbol(symbol_code("RAMCORE"), 4);
+        symbol ram_symbol = symbol(symbol_code("RAM"), 0);
+        uint64_t init_ram_amount = 10*1024;
+        // init stake net for new community account
+        asset init_net = asset(1'0000, this->core_symbol);
+        // init statke cpu for new community account
+        asset init_cpu = asset(1'0000, this->core_symbol);
+
```

Comment:
better to declare const for ramcore_symbol, ram_symbol if there are no update in the future.

This will need more code and reduce the performance